crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -602,6 +608,42 @@ async fn health_checks(State(connection): State<ConnectionConfig>) -> StatusCode
     }
 }
 
+#[derive(serde::Deserialize)]
+struct HealthParam {
+    max_checkpoint_ts_lag: Option<u64>,
```

Comment:
nit: add the unit in the name here?

```suggestion
    max_checkpoint_lag_ms: Option<u64>,
```

Reply:
Renamed